source/leuven/credits.rst

```diff
@@ -48,52 +44,130 @@ Please contact your VSC coordinator/contact or your :ref:`local support staff
+- ``walltime`` is the number of minutes that the job ran
 
-The Tier-2 cluster has several types of compute nodes, none of which
-is actually a reference node. The values for the different types are
-listed at the following links:
+.. note::
 
-- :ref:`Genius <running jobs on genius>`
+    *Effective* number of cores is not necessarily equal to what the user requests.
+    E.g. if a job requests a single core/task, but the full memory of a node on wICE,
+    then one node is blocked for such a job. Consequently, the effective number of cores
+    will be 72, instead of 1.
+
+    Similarly, on a node with 4 GPUs, if a user requests to use a single GPU, one-fourth
+    of cores will be assigned to the job. So, e.g. on wICE ``num_cores`` will be 18.
+
+.. note::
+
+    *Effective* number of GPUs deserves an explanation. If a user requests e.g. 2 GPUs
+    for a job, then ``num_gpus`` will be 2. However, in certain cases, we allow multiple
+    users to share a GPU (called sharding). In such cases, ``num_gpus`` could be a factor
+    between zero and one.
+
+.. note::
+
+    The Tier-2 cluster has several types of compute nodes.
+    Hence, different ``TRESBillingWeights`` applies to 
+    different resources on different partitions of Genius and wICE.
+    The difference in cost between different machines/processors reflects
+    the performance difference between those types of nodes.
+    For additional information, you may refer to the 
+    `HPC Service Catalog<https://icts.kuleuven.be/sc/onderzoeksgegevens/hpc_vsc_page>`_
+    (login required).
 
 The difference in cost between different machines/processors reflects
-the performance difference between those types of nodes. The total cost
-of a job will typically be the same on any compute nodes, but the
+the price-performance difference between those types of nodes. The total cost
+of a job will be comparable on any compute node, but the
 walltime will be different, depending on the performance of the nodes.
 
 In the examples below, you run your jobs on a ``skylake`` node, for which
-we charge 10 credits per hour.
+we charge 10 000 Slurm credits per hour.
 
 An example of a job running on multiple nodes and cores is given below::
 
-   $ qsub  -A lp_astrophysics_014  -l nodes=2:ppn=36:skylake  simulation_3415.pbs
+   $ sbatch -A lp_astrophysics_014 -M genius -N 2 -ntasks-per-node 36 simulation_3415.slurm
+
+For Genius thin nodes we have ``TRESBillingWeights=CPU=4.62963``.
+If this job finishes in 2.5 hours (i.e., walltime is 150 minutes), the user
+will be charged::
+
+   4.62963 * (2 * 36) * 150 = 50 000 credits
+
+
+Charging rates
+--------------
+
+The charge rate for the various node types of Genius and wICE are listed in the table
+below.  
+The reported cost is the number of Slurm credits needed per core/GPU per minute.
 
-If this job finished in 2.5 hours (i.e., walltime is 9000 seconds), the user
-will be charged:
++---------+-----------------+------------------------+
```

Comment:
Originally, this page contained how to calculate the required credits per each job. Once I converted the content to Slurm, then I had to mention the weights in the formula. Consequently, I also had to list the weights, too to complement the whole text, to show how many Slurm credits are needed in a job. That's why it has appeared here. 

The service catalog has a slightly different focus; it tells how much Slurm credits and money (in EUR) costs per using different resources. Then users can convert their pocket money into Slurm credits before filling in the credit request form.

Given the fact that these two information complement one another, the link to the service catalog is also provided on `credits.rst` page. 

With this, I think users can have all necessary information, each from a relevant resource with minimal overlap in content.

What do you think?

Reply:
My point is that these values are harder to interpret than what is in the service catalog, so they add little value since users can retrieve them themselves with the command shown earlier on this page. It is not necessarily bad that this table is included, but of course it has to be maintained when we update values or add a partition (for instance superdome).